docs/React.md

```diff
@@ -0,0 +1,30 @@
+# A brief guide to the React migration
+
+If you want get familiar with React, start with the [official site](reactjs.org). It has a nice [tutorial](https://reactjs.org/tutorial/tutorial.html) and [guide](https://reactjs.org/docs/hello-world.html).
+
+## What do we want to achieve?
+
+We want to have separate codebases for server and client. The client will be written in React.
+
+We'll replace the current AngularJS components one by one. Eventually, we'll drop the AngularJS and create a React project. 
+
+### Rationale
+
+- React is more attractive to developers.
+- AngularJS is being replaced by newer frameworks.
+- [MEAN is not maintained](https://github.com/Trustroots/trustroots/issues/638). 
```

Comment:
I'd add:

- React is simple and minimalistic; we won't risk getting stuck with a framework like  happened with AngularJS v1.

- React is not exotic; plenty of tooling and documentation available & lots of folks are already familiar with React.

- It has the potential to help with [React Native](https://github.com/Trustroots/trustroots-expo-mobile/) implementation of Trustroots. Not necessarily by re-using components but by allowing us to [_"learn once, write everywhere"_](https://www.youtube.com/watch?v=LIeqUPvh8qY).